client/starwhale/base/client/models/models.py

```diff
@@ -1392,7 +1392,7 @@ class OptionField(SwBaseModel):
     type: OptionType
     required: bool
     multiple: bool
-    value: Optional[str] = None
+    value: Optional[Dict[str, Any]] = None
```

Comment:
It seems that the server will parse it as a bool or list. Would there be any issues using a dict in Python?

Reply:
The value is unused in client now.
What's more uncomfortable is that java's obj is converted into dict when passing through pydantic.